.github/workflows/build.yml

```diff
@@ -120,15 +121,23 @@ jobs:
 
         PYTHONUNBUFFERED=TRUE pytest -v test_libcertifier_framework.py
 
-        PYTHONUNBUFFERED=TRUE pytest -v test_certifier_framework.py
+        # NOTE: Some test cases in test_certifier_framework.py need the Certifier Service
```

Comment:
The comments should be self-explanatory as to why I had to do this jugglery in build.xml workflow steps.